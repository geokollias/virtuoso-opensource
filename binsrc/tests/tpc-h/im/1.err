Starting Impala Shell without Kerberos authentication
Connected to ip-10-0-1-251.ec2.internal:21000
Server version: impalad version 2.1.3-cdh5 RELEASE (build 20816e26a150d20d2c92c470aa40b342521b243e)
Query: use TPCH
Query: select l_returnflag,
	l_linestatus,
	sum(l_quantity) as sum_qty,
	sum(l_extendedprice) as sum_base_price,
	sum(l_extendedprice * (1 - l_discount)) as sum_disc_price,
	sum(l_extendedprice * (1 - l_discount) * (1 + l_tax)) as sum_charge,
	avg(l_quantity) as avg_qty,
	avg(l_extendedprice) as avg_price,
	avg(l_discount) as avg_disc,
	count(*) as count_order
from
	lineitem
where
	l_shipdate <= date_add('1998-12-01', -90)
group by
	l_returnflag,
	l_linestatus
order by
	l_returnflag,
	l_linestatus
Fetched 4 row(s) in 293.77s
