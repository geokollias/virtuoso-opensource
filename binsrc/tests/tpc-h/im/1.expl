[?1034h+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| Explain String                                                                                                                                                                                                                                                     |
+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| Estimated Per-Host Requirements: Memory=744.00MB VCores=2                                                                                                                                                                                                          |
| WARNING: The following tables are missing relevant table and/or column statistics.                                                                                                                                                                                 |
| tpch.lineitem                                                                                                                                                                                                                                                      |
|                                                                                                                                                                                                                                                                    |
| 05:MERGING-EXCHANGE [UNPARTITIONED]                                                                                                                                                                                                                                |
| |  order by: l_returnflag ASC, l_linestatus ASC                                                                                                                                                                                                                    |
| |  hosts=1 per-host-mem=unavailable                                                                                                                                                                                                                                |
| |  tuple-ids=3 row-size=66B cardinality=60003790                                                                                                                                                                                                                   |
| |                                                                                                                                                                                                                                                                  |
| 02:SORT                                                                                                                                                                                                                                                            |
| |  order by: l_returnflag ASC, l_linestatus ASC                                                                                                                                                                                                                    |
| |  hosts=1 per-host-mem=176.00MB                                                                                                                                                                                                                                   |
| |  tuple-ids=3 row-size=66B cardinality=60003790                                                                                                                                                                                                                   |
| |                                                                                                                                                                                                                                                                  |
| 04:AGGREGATE [FINALIZE]                                                                                                                                                                                                                                            |
| |  output: sum:merge(l_quantity), sum:merge(l_extendedprice), sum:merge(l_extendedprice * (1 - l_discount)), sum:merge(l_extendedprice * (1 - l_discount) * (1 + l_tax)), avg:merge(l_quantity), avg:merge(l_extendedprice), avg:merge(l_discount), count:merge(*) |
| |  group by: l_returnflag, l_linestatus                                                                                                                                                                                                                            |
| |  hosts=1 per-host-mem=128.00MB                                                                                                                                                                                                                                   |
| |  tuple-ids=2 row-size=66B cardinality=60003790                                                                                                                                                                                                                   |
| |                                                                                                                                                                                                                                                                  |
| 03:EXCHANGE [HASH(l_returnflag,l_linestatus)]                                                                                                                                                                                                                      |
| |  hosts=1 per-host-mem=0B                                                                                                                                                                                                                                         |
| |  tuple-ids=1 row-size=66B cardinality=60003790                                                                                                                                                                                                                   |
| |                                                                                                                                                                                                                                                                  |
| 01:AGGREGATE                                                                                                                                                                                                                                                       |
| |  output: sum(l_quantity), sum(l_extendedprice), sum(l_extendedprice * (1 - l_discount)), sum(l_extendedprice * (1 - l_discount) * (1 + l_tax)), avg(l_quantity), avg(l_extendedprice), avg(l_discount), count(*)                                                 |
| |  group by: l_returnflag, l_linestatus                                                                                                                                                                                                                            |
| |  hosts=1 per-host-mem=128.00MB                                                                                                                                                                                                                                   |
| |  tuple-ids=1 row-size=66B cardinality=60003790                                                                                                                                                                                                                   |
| |                                                                                                                                                                                                                                                                  |
| 00:SCAN HDFS [tpch.lineitem, RANDOM]                                                                                                                                                                                                                               |
|    partitions=1/1 files=82 size=20.20GB                                                                                                                                                                                                                            |
|    predicates: l_shipdate <= date_add('1998-12-01', -90)                                                                                                                                                                                                           |
|    table stats: 600037902 rows total                                                                                                                                                                                                                               |
|    columns missing stats: l_returnflag, l_linestatus                                                                                                                                                                                                               |
|    hosts=1 per-host-mem=616.00MB                                                                                                                                                                                                                                   |
|    tuple-ids=0 row-size=50B cardinality=60003790                                                                                                                                                                                                                   |
+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
