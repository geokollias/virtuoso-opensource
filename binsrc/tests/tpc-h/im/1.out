[?1034h+--------------+--------------+------------+-------------------+-------------------+-------------------+-------------------+-------------------+---------------------+-------------+
| l_returnflag | l_linestatus | sum_qty    | sum_base_price    | sum_disc_price    | sum_charge        | avg_qty           | avg_price         | avg_disc            | count_order |
+--------------+--------------+------------+-------------------+-------------------+-------------------+-------------------+-------------------+---------------------+-------------+
| A            | F            | 3775127758 | 5660776097198.913 | 5377736398182.777 | 5592847429513.244 | 25.49937042327543 | 38236.11698433504 | 0.05000224347713225 | 148047881   |
| N            | F            | 98553062   | 147771098385.9689 | 140384965965.0348 | 145999793032.7831 | 25.50155695688288 | 38237.19938880162 | 0.04998528433773638 | 3864590     |
| N            | O            | 7436302976 | 11150725681374.18 | 10593195308232.23 | 11016932248184.29 | 25.50000940437419 | 38237.22764636298 | 0.04999791823830614 | 291619617   |
| R            | F            | 3775724970 | 5661603032747.248 | 5378513563915.351 | 5593662252666.573 | 25.50006628406532 | 38236.69725846585 | 0.05000130428586257 | 148067261   |
+--------------+--------------+------------+-------------------+-------------------+-------------------+-------------------+-------------------+---------------------+-------------+
